### ISSUE TITLE
Cannot import ndcurves

### ISSUE THREAD

**Issue by robopassio** [loco-3d/ndcurves#60]
After installing inside catkin workspace (no error). I cannot run this command in python.
```
ImportError: No module named ndcurves
```
Im using ros melodic, ubuntu 18.04

**Comment by robopassio** [comment]
After several efforts I got:
```
>>> import ndcurves
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/my_pc/catkin_curves/devel_isolated/ndcurves/lib/python2.7/dist-packages/ndcurves/__init__.py", line 5, in <module>
    from .ndcurves import *  # noqa
ImportError: /home/my_pc/catkin_curves/devel_isolated/ndcurves/lib/python2.7/dist-packages/ndcurves/ndcurves.so: undefined symbol: _ZN5boost7archive17xml_iarchive_implINS0_12xml_iarchiveEE13load_overrideERNS0_15class_name_typeEi
```

**Comment by nim65s** [comment]
Hi @robopassio,

I missed your issue, sorry for this. But meanwhile, it was fixed by #61.
